usmqe_tests/rpm/test_rpm.py

```diff
@@ -106,9 +106,18 @@ def test_rpmdeplint(rpm_package, check_command, tendrl_repos, centos_repos):
     cp = subprocess.run(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
     LOGGER.debug("STDOUT: %s", cp.stdout)
     LOGGER.debug("STDERR: %s", cp.stderr)
-    check_msg = "rpmdeplint return code should be 0 indicating no errors"
-    pytest.check(cp.returncode == 0, msg=check_msg)
+    # check_msg = "rpmdeplint return code should be 0 indicating no errors"
+    # pytest.check(cp.returncode == 0, msg=check_msg)
```

Comment:
I would rather drop these lines completely instead of commenting them out - if anyone needs the lines, he could use git.